qiskit/test/mock/utils/fake_backend_builder.py

```diff
@@ -0,0 +1,331 @@
+
+    def build_conf(self) -> PulseBackendConfiguration:
+        """Build configuration for backend."""
+        h_str = [
+            ",".join(["_SUM[i,0,{n_qubits}".format(n_qubits=self.n_qubits),
+                      "wq{i}/2*(I{i}-Z{i})]"]),
+            ",".join(["_SUM[i,0,{n_qubits}".format(n_qubits=self.n_qubits),
+                      "omegad{i}*X{i}||D{i}]"])
+        ]
+        variables = []
+        for (qubit1, qubit2) in self.cmap:
+            h_str += [
+                "jq{q1}q{q2}*Sp{q1}*Sm{q2}".format(q1=qubit1, q2=qubit2),
+                "jq{q1}q{q2}*Sm{q1}*Sp{q2}".format(q1=qubit1, q2=qubit2)
+            ]
+
+            variables.append(("jq{q1}q{q2}".format(q1=qubit1, q2=qubit2), 0))
+        for i, (qubit1, qubit2) in enumerate(self.cmap):
+            h_str.append("omegad{0}*X{1}||U{2}".format(qubit1, qubit2, i))
+        for i in range(self.n_qubits):
+            variables += [
+                ("omegad{}".format(i), 0),
+                ("wq{}".format(i), 0)
+            ]
+        hamiltonian = {
+            'h_str': h_str,
+            'description': 'Hamiltonian description for {} qubits backend.'.format(self.n_qubits),
+            'qub': {i: 2 for i in range(self.n_qubits)},
+            'vars': dict(variables)
+        }
+
+        meas_map = [list(range(self.n_qubits))]
+        qubit_lo_range = [[self.qubit_frequency - .5, self.qubit_frequency + .5]
+                          for _ in range(self.n_qubits)]
+        meas_lo_range = [[6.5, 7.5] for _ in range(self.n_qubits)]
+        u_channel_lo = [[UchannelLO(q=i, scale=1. + 0.j)] for i in range(len(self.cmap))]
+
+        return PulseBackendConfiguration(
+            backend_name=self.name,
+            backend_version=self.version,
+            n_qubits=self.n_qubits,
+            meas_levels=[0, 1, 2],
+            basis_gates=self.basis_gates,
+            simulator=False,
+            local=True,
+            conditional=True,
+            open_pulse=True,
+            memory=False,
+            max_shots=65536,
+            gates=[],
+            coupling_map=self.cmap,
+            n_registers=self.n_qubits,
+            n_uchannels=self.n_qubits,
+            u_channel_lo=u_channel_lo,
+            meas_level=[1, 2],
+            qubit_lo_range=qubit_lo_range,
+            meas_lo_range=meas_lo_range,
+            dt=1.3333,
+            dtm=10.5,
+            rep_times=[1000],
+            meas_map=meas_map,
+            channel_bandwidth=[],
+            meas_kernels=['kernel1'],
+            discriminators=['max_1Q_fidelity'],
+            acquisition_latency=[],
+            conditional_latency=[],
+            hamiltonian=hamiltonian
+        )
+
+    def build_defaults(self) -> PulseDefaults:
+        """Build backend defaults."""
+
+        qubit_freq_est = np.linspace(4.9, 5.1, self.n_qubits).tolist()
+        meas_freq_est = np.linspace(6.4, 6.6, self.n_qubits).tolist()
+        buffer = 10
```

Comment:
This has been deprecated. I think it still has to be supplied, but let's just set it to `0`

Reply:
https://github.com/Qiskit/qiskit-terra/pull/3797/commits/a66464df5a1c1ac8eaff73dff1415cfcf71a9ebb